clients/web/src/models/UserModel.js

```diff
@@ -1,6 +1,26 @@
 girder.models.UserModel = girder.Model.extend({
     resourceName: 'user',
 
+    // Sets this model to the currently logged in user; has no effect is no user
+    // is logged in.
+    current: function (success, error) {
+        girder.restRequest({
+            method: 'GET',
+            path: '/user/me'
+        }).then(_.bind(function (user) {
+            if (user) {
+                this.set(user);
+                if (success) {
+                    success(this);
+                }
+            } else {
+                if (error) {
+                    error(this);
+                }
```

Comment:
I don't think we want to treat this as an error condition since `/user/me` is meant as a way to test whether or not you're logged in. It makes it a bit tricky since you put this in the model layer; maybe it makes more sense to move it out under the `girder` object like the other two functions you added?